ffi/src/std_impls.rs

```diff
@@ -19,16 +15,14 @@ use crate::{
 // NOTE: This can be contested as it is nowhere documented that String is
 // actually transmutable into Vec<u8>, but implicitly it should be
 // SAFETY: String type should be transmutable into Vec<u8>
-ffi_type! {unsafe impl Transparent for String[Vec<u8>] validated with {|target| core::str::from_utf8(target).is_ok()} }
+ffi_type! {unsafe impl Transparent for String[Vec<u8>] {|target| core::str::from_utf8(target).is_ok()} }
 // NOTE: `core::str::as_bytes` uses transmute internally which means that
 // even though it's a string slice it can be transmuted into byte slice.
-ffi_type! {unsafe impl<'slice> Transparent for &'slice str[&'slice [u8]] validated with {|target| core::str::from_utf8(target).is_ok()} }
+ffi_type! {unsafe impl<'slice> Transparent for &'slice str[&'slice [u8]] {|target| core::str::from_utf8(target).is_ok()} }
```

Comment:
I like the old way better. It's self explanatory. This way it doesn't even really look like a lambda, so I don't immediately know what I'm supposed to do.

Reply:
I like the old way better as well but the macro rules didn't allow me to keep it like that after fixing the bug. I'll try to restore this

Reply:
fixed